### ISSUE TITLE
DBG_PRINTF issue

### ISSUE THREAD

**Issue by larseggert** [private-octopus/picoquic#606]
When compiling with `clang`, I get a gazillion warnings due to an issue with the `DBG_PRINTF` macro:

```
../lib/deps/picoquic/picoquictest/tls_api_test.c:6036:13: warning: adding 'unsigned long' to a string does not append to the string [-Wstring-plus-int]
            DBG_PRINTF("%s", "Client addresses were not properly documented\n");
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../lib/deps/picoquic/picoquic/util.h:77:18: note: expanded from macro 'DBG_PRINTF'
        __FILE__ + MAX(DBG_PRINTF_FILENAME_MAX, sizeof(__FILE__)) - DBG_PRINTF_FILENAME_MAX, \
        ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../lib/deps/picoquic/picoquictest/tls_api_test.c:6036:13: note: use array indexing to silence this warning
```

**Comment by huitema** [comment]
@larseggert That's weird, because the automated tests using clang on Linux and IOS are passing fine. What compile options are you using?

**Comment by huitema** [comment]
@larseggert can you check whether PR #607 addresses the issue?

**Comment by huitema** [comment]
Fixed in PR #607
